moveit_core/constraint_samplers/src/constraint_sampler_tools.cpp

```diff
@@ -59,13 +61,17 @@ double constraint_samplers::countSamplesPerSecond(const ConstraintSamplerPtr& sa
 {
   if (!sampler)
   {
-    ROS_ERROR_NAMED("constraint_samplers", "No sampler specified for counting samples per second");
+    RCLCPP_ERROR(LOGGER, "No sampler specified for counting samples per second");
     return 0.0;
   }
   robot_state::RobotState ks(reference_state);
   unsigned long int valid = 0;
   unsigned long int total = 0;
-  ros::WallTime end = ros::WallTime::now() + ros::WallDuration(1.0);
+  //TODO RCLCPP::WallTimer has not a .now() function, and there is no wallDuration (they use std::chrono::nanoseconds)
+  // I'll use rclcpp::time, but this is something to fix.
+  rclcpp::Duration duration(1,0);
```

Comment:
I think this solution is good. We could instantiate `rclcpp::Duration` inline, though.

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/c8fb7c101161e7d23b1caf4baa2964c04f6f7ee8